docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.9"
+services:
+  bulletin:
+    build:
+      context: .
+      dockerfile: Dockerfile.release
+    env_file: .env
+    environment:
+      - RAILS_LOG_TO_STDOUT=1
+      - RAILS_SERVE_STATIC_FILES=1
+      - RACK_ENV=production
+      - RAILS_ENV=production
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+    volumes:
+      - ./docker-entrypoint.sh:/docker-entrypoint.sh
+    entrypoint: /docker-entrypoint.sh
+    command: ""
+    ports:
+      - "8000:8000"
+    depends_on:
+      - redis
```

Comment:
As this is for production, shouldn't we add `    restart: unless-stopped` for all the services?

Reply:
Sure, although I imagine a real production docker-compose pulling a image directly from dockerhub instead of building it here. Let's say that this also work for development

Reply:
Also, production configurations can be very personal. I imagine most people would use a Nginx service integrated here instead of exposing the 8000 port form rails.